regression-test/suites/query_p0/sql_functions/array_functions/test_array_functions.groovy

```diff
@@ -70,4 +70,32 @@ suite("test_array_functions") {
     qt_select "SELECT k1, array_popback(k5) from ${tableName} ORDER BY k1"
     qt_select "SELECT k1, array_popback(k6) from ${tableName} ORDER BY k1"
     qt_select "SELECT k1, array_popback(k7) from ${tableName} ORDER BY k1"
+
+    def tableName2 = "tbl_test_array_range"
+    sql """DROP TABLE IF EXISTS ${tableName2}"""
+    sql """
+            CREATE TABLE IF NOT EXISTS ${tableName2} (
+              `k1` int(11) NULL COMMENT "",
+              `k2` int(11) NULL COMMENT "",
+              `k3` int(11) NULL COMMENT ""
+            ) ENGINE=OLAP
+            DUPLICATE KEY(`k1`)
+            DISTRIBUTED BY HASH(`k1`) BUCKETS 1
+            PROPERTIES (
+            "replication_allocation" = "tag.location.default: 1",
+            "storage_format" = "V2"
+            )
+        """
+    sql """ INSERT INTO ${tableName2} VALUES(1,3,5) """
```

Comment:
Could you also add some abnormal cases? such as:
1. end > start;
2. step > (end - start);
3. negative start, end and step;

Reply:
done